assets/js/worker.js

```diff
@@ -1,18 +1,38 @@
-onmessage = function(event) {
-	self.importScripts('https://unpkg.com/lunr/lunr.js');
-	
+let scriptLoaded = false;
+
+onmessage = async function(event) {
+	const scriptUrl = 'https://unpkg.com/lunr/lunr.js';
+  const integrityHash = 'sha256-lDFybwXA6uKm5U3Bl3CUIoafJcrUTyQw0vt92ugMxxc=';
+	if (scriptLoaded) return
+
+	let response
+	if (!scriptLoaded) {
```

Comment:
This `if` wrapper is not needed, since you have the early return `if (scriptLoaded) return` earlier.